src/constants/auth_example.json

```diff
@@ -0,0 +1,12 @@
+{
+    "token": {
+        "discord_app": "discord app token",
+        "discord": "discord token",
+        "client_id": "client id",
+        "pub_key": "supersecretkey"
+    },
+    "apiStorage": {
+        "username": "username",
+        "password": "password"
+    }
+}
```

Comment:
Then current auth file is


{
    "token": {
        "discord": "discord_token",
        "giphy": "giphy_token"
    }
}